### PR TITLE
Add Dockerfiles for tests on PHP 7.3

### PR DESCRIPTION
**Pull request by danxuliu**
Similar to #171 this is just to be prepared when 7.1 is dropped in Nextcloud server.

@rullzer Please setup the needed Docker repositories to be able to tag the images ;-)

Note that the Samba native image depends on the Samba non native image, so it can not be tagged until that image is created.

### CODE REVIEW

File: samba-native-php7.3/smbclient.ini [hunk 1]
```diff
@@ -0,0 +1 @@
+extension="smbclient.so"
```
Comment (skjnldsv) [inline samba-native-php7.3/smbclient.ini]:
```suggestion
extension=smbclient
```
I think they're usually written without the extension and without quotes :)

Comment (danxuliu) [reply samba-native-php7.3/smbclient.ini]: No idea, I just copy pasted from the [samba-native-php7.1 files](https://github.com/nextcloud/docker-ci/blob/322c63de48793d4e1b358f893180123ade1fcdc3/samba-native-php7.1/smbclient.ini) :man_shrugging: 

I have checked how they appear in other ini files in the PHP 7.3 Docker image and this is the result:
```
...]# docker run --rm nextcloudci/php7.3:php7.3-3 find /etc/php/7.3/cli/conf.d/ -type f -exec grep -Hn extension {} \;
/etc/php/7.3/cli/conf.d/20-ast.ini:1:extension=ast.so
```

Not a very big sample, unfortunately :-P So as you wish, no quotes and no extension, no quotes but extension, quotes and extension... I have no preference ;-)

Comment (skjnldsv) [reply samba-native-php7.3/smbclient.ini]: I really do not care xD
If it works, good for me :D

**Comment by rullzer** [comment]
Done:
* Integration: 
  - https://hub.docker.com/repository/docker/nextcloudci/integration-php7.3/builds
* Litmus:
  - https://hub.docker.com/repository/docker/nextcloudci/litmus-7.3/builds

Red:
* memcached:
  - https://hub.docker.com/repository/docker/nextcloudci/php7.3-memcached/builds
* samba-non-native
  - https://hub.docker.com/repository/docker/nextcloudci/samba-non-native-php7.3/builds
* samba-native:
  - todo
* webdav apache
  - https://hub.docker.com/repository/docker/nextcloudci/webdav-apache-php7.3/builds

**Comment by rullzer** [comment]
So I started being pragmatic because locallybuilding and puhsing was faster and more reliable

**Pull request merged**